2023-05-25 16:19:35,855 INFO    MainThread:717500 [wandb_setup.py:_flush():76] Current SDK version is 0.15.3
2023-05-25 16:19:35,855 INFO    MainThread:717500 [wandb_setup.py:_flush():76] Configure stats pid to 717500
2023-05-25 16:19:35,855 INFO    MainThread:717500 [wandb_setup.py:_flush():76] Loading settings from /home/edo/.config/wandb/settings
2023-05-25 16:19:35,856 INFO    MainThread:717500 [wandb_setup.py:_flush():76] Loading settings from /home/edo/Desktop/orc/03_assignment/wandb/settings
2023-05-25 16:19:35,856 INFO    MainThread:717500 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'_executable': '/usr/bin/python3'}
2023-05-25 16:19:35,856 INFO    MainThread:717500 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'DQN template.py', 'program': 'DQN template.py'}
2023-05-25 16:19:35,856 INFO    MainThread:717500 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2023-05-25 16:19:35,856 INFO    MainThread:717500 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2023-05-25 16:19:35,856 INFO    MainThread:717500 [wandb_init.py:_log_setup():507] Logging user logs to /home/edo/Desktop/orc/03_assignment/wandb/run-20230525_161935-soeidyxs/logs/debug.log
2023-05-25 16:19:35,856 INFO    MainThread:717500 [wandb_init.py:_log_setup():508] Logging internal logs to /home/edo/Desktop/orc/03_assignment/wandb/run-20230525_161935-soeidyxs/logs/debug-internal.log
2023-05-25 16:19:35,856 INFO    MainThread:717500 [wandb_init.py:init():547] calling init triggers
2023-05-25 16:19:35,856 INFO    MainThread:717500 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {}
2023-05-25 16:19:35,856 INFO    MainThread:717500 [wandb_init.py:init():596] starting backend
2023-05-25 16:19:35,856 INFO    MainThread:717500 [wandb_init.py:init():600] setting up manager
2023-05-25 16:19:35,869 INFO    MainThread:717500 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-05-25 16:19:36,035 INFO    MainThread:717500 [wandb_init.py:init():606] backend started and connected
2023-05-25 16:19:36,040 INFO    MainThread:717500 [wandb_init.py:init():700] updated telemetry
2023-05-25 16:19:36,057 INFO    MainThread:717500 [wandb_init.py:init():737] communicating run to backend with 60.0 second timeout
2023-05-25 16:19:36,336 WARNING MainThread:717500 [wandb_init.py:init():1172] interrupted
Traceback (most recent call last):
  File "/home/edo/.local/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 1150, in init
    run = wi.init()
  File "/home/edo/.local/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 740, in init
    result = run_init_handle.wait(
  File "/home/edo/.local/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "/home/edo/.local/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "/home/edo/.local/lib/python3.8/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "/usr/lib/python3.8/threading.py", line 558, in wait
    signaled = self._cond.wait(timeout)
  File "/usr/lib/python3.8/threading.py", line 306, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
2023-05-25 16:19:38,821 WARNING MsgRouterThr:717500 [router.py:message_loop():77] message_loop has been closed
